boards/digilent/zybo/zybo_defconfig

```diff
@@ -4,7 +4,6 @@
 CONFIG_ARM_ARCH_TIMER=y
 CONFIG_SYS_CLOCK_HW_CYCLES_PER_SEC=325000000
 
-CONFIG_PINCTRL=y
```

Comment:
For FPGA-based designs like this, it actually is up to the board to select pinctrl if needed by the design. Some designs route every pin in the FPGA, som do it at runtime.

Reply:
This needs to stay as pinctrl is optional in `drivers/serial/uart_xlnx_ps.c`.